application/comit_node/src/swap_protocols/rfc003/state_machine.rs

```diff
@@ -535,91 +515,3 @@ impl_display!(AlphaRefundedBetaFunded);
 impl_display!(AlphaFundedBetaRedeemed);
```

Comment:
Do we still need that? I think we don't :)

Reply:
We still log the state transition using these. Should that change?